ui/src/style/paper-summary.less

```diff
@@ -44,3 +44,20 @@
     color: deepskyblue;
   }
 }
+
+/**
+ * Actions
+ */
+
+@{mui-button-base} {
+  @icon-size: 1.8rem;
+
+  &.paper-summary__action {
+    color: #8c9296;
```

Comment:
nit: should this be a variable in variables.less? :o

Reply:
Good call! I'm actually expanding this PR a bit, and it's been converted to a variable in the extra changes.